test/e2e/nginx/conf.d/nginx.conf

```diff
@@ -71,32 +70,14 @@ http {
             }
         }
 
-        location /tier2/lb {
-            default_type text/html;
-
-            rewrite_by_lua_block {
-                skywalking_tracer:start("e2e-test-with-mock-collector:upstream_ip2:port2")
-            }
-
-            proxy_pass http://127.0.0.1:8080/backend;
-
-            body_filter_by_lua_block {
-                skywalking_tracer:finish()
-            }
-
-            log_by_lua_block {
-                skywalking_tracer:prepareForReport()
-            }
-        }
         location /suffix {
-
             default_type text/html;
             content_by_lua_block {
                 ngx.say("<p>Suffix for testing only.</p>")
             }
 
             rewrite_by_lua_block {
-                skywalking_tracer:start("e2e-test-with-mock-collector:upstream_ip:port")
+                skywalking_tracer:start("172.16.238.13:9092")
```

Comment:
Why this is changed to a real IP?

Reply:
Since VLAN and IP are set in the docker-compose of E2E, the IP of each container is fixed. Changing to IP here will show some real use cases.

Reply:
reverted